Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -2276,9 +2274,38 @@ void WindowManager::apply_cursor_theme(String const& theme_name)
 
     Compositor::the().invalidate_cursor();
     m_config->write_entry("Mouse", "CursorTheme", theme_name);
+    verify_config_saved();
+}
+
+void WindowManager::set_cursor_highlight_radius(int radius)
+{
+    // TODO: Validate radius
+    m_cursor_highlight_radius = radius;
+    Compositor::the().invalidate_cursor();
+    m_config->write_num_entry("Mouse", "CursorHighlightRadius", radius);
+    verify_config_saved();
+}
+
+void WindowManager::set_cursor_highlight_color(Gfx::Color const& color)
+{
+    m_cursor_highlight_color = color;
+    Compositor::the().invalidate_cursor();
+    m_config->write_color_entry("Mouse", "CursorHighlightColor", color);
+    verify_config_saved();
+}
+
+bool WindowManager::is_cursor_highlight_enabled() const
+{
+    return m_cursor_highlight_radius > 0 && m_cursor_highlight_color.alpha() > 0;
+}
+
+bool WindowManager::verify_config_saved()
```

Comment:
How about renaming this to `WindowManager::save_config` instead? This name implies that it can't fail and that it actually already happened.

Reply:
I don't think it's really saving the config either, what about `WindowManager::sync_config_to_disk ()`?

Reply:
I'll just go with that :)